spec/decorators/order_detail_decorator_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.describe OrderDetailDecorator, type: :decorator do
+  let(:decorated) { OrderDetailDecorator.new(order_detail) }
+
+  describe "delegates to assigned_user" do
+    let(:order_detail) do
+      OrderDetail.new.tap do |order_detail|
```

Comment:
You could actually use `object_double(OrderDetail.new, assigned_user: :assigned_user)`. I think you could also use `instance_double`. I don't fully understand the difference.